simtools/simtel/simtel_runner.py

```diff
@@ -197,6 +217,11 @@ def _raise_simtel_error(self):
     def _run_simtel_and_check_output(self, command):
         """
         Run the sim_telarray command and check the exit code.
+
+        Raises
+        ------
+        SimtelExecutionError
+            if output of command is 0.
```

Comment:
Maybe better to speak about success or failure here, as we don't test for output `0` here.